gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/resources/openapi/management-openapi-v2.yaml

```diff
@@ -423,6 +423,36 @@ paths:
                     description: API successfully stopped
                 default:
                     $ref: "#/components/responses/Error"
+    /environments/{envId}/apis/{apiId}/_transfer-ownership:
+        parameters:
+            - $ref: "#/components/parameters/envIdParam"
+            - $ref: "#/components/parameters/apiIdParam"
+        post:
+            tags:
+                - APIs
+            summary: Transfer the ownership of the API
+            description: |-
+                Transfer the ownership of the API to a user, a group or an api member.
+                
+                Return a 404 HTTP Error if API cannot be found.
+
+                Return a 400 HTTP Error:
+                 - when user tries to stop an ARCHIVED API
+                 - when the API is already STOPPED.
+
+                User must have the API_MEMBER[UPDATE] permission.
+            operationId: transferOwnership
+            requestBody:
+                content:
+                    application/json:
+                        schema:
+                            $ref: "#/components/schemas/ApiTransferOwnership"
+                required: true
+            responses:
+                "200":
+                    description: Ownership has been transfered successfully
```

Comment:
Is it truly that the transfer is successful? Or is it just that the demand to transfer the ownership is done? Since the service call is void in ApiResource, it doesn't seem very obvious to me.

Reply:
From what I can see, the transfer ownership method seems pretty synchronous...